superset/migrations/versions/d3822549a380_refactor_alerting_framework.py

```diff
@@ -0,0 +1,104 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""refactor_alerting_framework
+
+Revision ID: d3822549a380
+Revises: f80a3b88324b
+Create Date: 2020-08-27 14:20:24.477024
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "d3822549a380"
+down_revision = "f80a3b88324b"
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import mysql
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "alert_validators",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("name", sa.String(length=150), nullable=False),
+        sa.Column(
+            "validator_type",
```

Comment:
can you double check if we use strings vs enum types for the migrations ?
there are pros and cons for both, curious what's a common way in this codebase